fuel-asm/src/args.rs

```diff
@@ -50,218 +50,218 @@ crate::enum_try_from! {
-        InputMessageNonce = 0x118,
+        InputMessageNonce = 0x243,
 
         /// Set `$rA` to `tx.inputs[$rB].witnessIndex`
-        InputMessageWitnessIndex = 0x119,
+        InputMessageWitnessIndex = 0x244,
 
         /// Set `$rA` to `tx.inputs[$rB].dataLength`
-        InputMessageDataLength = 0x11A,
+        InputMessageDataLength = 0x245,
 
         /// Set `$rA` to `tx.inputs[$rB].predicateLength`
-        InputMessagePredicateLength = 0x11B,
+        InputMessagePredicateLength = 0x246,
 
         /// Set `$rA` to `tx.inputs[$rB].predicateDataLength`
-        InputMessagePredicateDataLength = 0x11C,
+        InputMessagePredicateDataLength = 0x247,
 
         /// Set `$rA` to `Memory address of tx.inputs[$rB].data`
-        InputMessageData = 0x11D,
+        InputMessageData = 0x248,
 
         /// Set `$rA` to `Memory address of tx.inputs[$rB].predicate`
-        InputMessagePredicate = 0x11E,
+        InputMessagePredicate = 0x249,
 
         /// Set `$rA` to `Memory address of tx.inputs[$rB].predicateData`
-        InputMessagePredicateData = 0x11F,
+        InputMessagePredicateData = 0x24A,
 
         /// Set `$rA` to `Memory address of tx.inputs[$rB].predicateGasUsed`
-        InputMessagePredicateGasUsed = 0x120,
+        InputMessagePredicateGasUsed = 0x24B,
 
         /// Set `$rA` to `tx.outputs[$rB].type`
-        OutputType = 0x201,
+        OutputType = 0x300,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].to`
-        OutputCoinTo = 0x202,
+        OutputCoinTo = 0x301,
 
         /// Set `$rA` to `tx.outputs[$rB].amount`
-        OutputCoinAmount = 0x203,
+        OutputCoinAmount = 0x302,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].asset_id`
-        OutputCoinAssetId = 0x204,
+        OutputCoinAssetId = 0x303,
 
         /// Set `$rA` to `tx.outputs[$rB].inputIndex`
-        OutputContractInputIndex = 0x205,
+        OutputContractInputIndex = 0x304,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].balanceRoot`
-        OutputContractBalanceRoot = 0x206,
+        OutputContractBalanceRoot = 0x305,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].stateRoot`
-        OutputContractStateRoot = 0x207,
+        OutputContractStateRoot = 0x306,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].contractID`
-        OutputContractCreatedContractId = 0x208,
+        OutputContractCreatedContractId = 0x307,
 
         /// Set `$rA` to `Memory address of tx.outputs[$rB].stateRoot`
-        OutputContractCreatedStateRoot = 0x209,
+        OutputContractCreatedStateRoot = 0x308,
 
         /// Set `$rA` to `tx.witnesses[$rB].dataLength`
-        WitnessDataLength = 0x301,
+        WitnessDataLength = 0x400,
 
         /// Set `$rA` to `Memory address of tx.witnesses[$rB].data`
-        WitnessData = 0x302,
+        WitnessData = 0x401,
+
+        /// Set `$rA` to `tx.policies[$rB].policyTypes`
+        PolicyTypes = 0x501,
```

Comment:
We should bump these up since we aren't using `GTF_POLICY_COUNT` anymore

```suggestion
        PolicyTypes = 0x500,
```